api.swagger.yaml

```diff
@@ -14963,7 +14964,7 @@ components:
                   format: uuid
                   example: '01234567-89ab-cdef-1234-56789abcdef0'
     list-list-site-section-schema:
-      allOf:
```

Comment:
This should be allOf it is combining the common schema with the specific object. anyOf would mean it can be either or both.